bin/cli.js

```diff
@@ -76,11 +81,21 @@ require('yargs')
             args: argv.puppeteerArgs ? argv.puppeteerArgs.split(' ') : [],
             executablePath: argv.puppeteerExecutablePath,
         };
+
+        if (debug) {
+          launchArgs.headless = false;
+        }
+
         const browser = await puppeteer.launch(launchArgs);
 
         try {
           const page = await browser.newPage();
 
+          if (debug) {
+            page.bringToFront();
+            page.on('console', msg => console.log('PAGE LOG:', msg.text()));
```

Comment:
I wonder, should we just *always* forward console messages to the terminal? Can't think of a reason why not 🤔

Reply:
Doesn't have to block this PR though, we can always revisit this later.

Reply:
I guess it can get noisy in the terminal unless you're trying to debug something, so under this flag made sense to me.